src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneIndexWriter.cs

```diff
@@ -106,7 +106,7 @@ private void CreateIndexWriter()
             using (indexWriter.MergeScheduler)
             {
             }
-            indexWriter.SetMergeScheduler(new ErrorLoggingConcurrentMergeScheduler());
+            indexWriter.SetMergeScheduler(new SerialMergeScheduler());
```

Comment:
I'm actually fine with it being serial, we just need to be able to track when this is happening and add it to the metrics.